python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/port-channel-interfaces.j2

```diff
@@ -28,9 +18,42 @@ interface {{ port_channel_interface.name }}
 {%     if port_channel_interface.mtu is arista.avd.defined %}
    mtu {{ port_channel_interface.mtu }}
 {%     endif %}
+{%     if port_channel_interface.logging.event.link_status is arista.avd.defined(true) %}
+   logging event link-status
+{%     elif port_channel_interface.logging.event.link_status is arista.avd.defined(false) %}
+   no logging event link-status
+{%     endif %}
 {%     if port_channel_interface.bgp.session_tracker is arista.avd.defined %}
    bgp session tracker {{ port_channel_interface.bgp.session_tracker }}
 {%     endif %}
+{%     if port_channel_interface.l2_protocol.forwarding_profile is arista.avd.defined %}
+   l2-protocol forwarding profile {{ port_channel_interface.l2_protocol.forwarding_profile }}
+{%     endif %}
+{%     if port_channel_interface.vlans is arista.avd.defined and port_channel_interface.mode is arista.avd.defined and port_channel_interface.mode in ["access", "dot1q-tunnel"] %}
+   switchport access vlan {{ port_channel_interface.vlans }}
+{%     endif %}
+{%     if port_channel_interface.vlans is arista.avd.defined and port_channel_interface.mode is arista.avd.defined("trunk") %}
+   switchport trunk allowed vlan {{ port_channel_interface.vlans }}
```

Comment:
This will go below `switchport vlan translation`

```
   switchport vlan translation out required
   switchport trunk allowed vlan 110-111,210-211
   switchport mode trunk
```

Reply:
Fixed. Thanks.